### PR TITLE
Fix special consideration packages validation

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR:
* removes validation regarding special handling modules
* add validation regarding special consideration packages

Closes OSIDB-2955

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
LGTM

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix external sync order in serializers (OSIDB-3029)
 - Make Taskman service validate Jira token (OSIDB-2203)
 
+### Changed
+- special_handling_flaw_missing_cve_description Alert to
+  special_consideration_flaw_missing_cve_description (OSIDB-2955)
+- special_handling_flaw_missing_statement Alert to
+  special_consideration_flaw_missing_statement (OSIDB-2955)
```
Comment (jobselko) [inline docs/CHANGELOG.md:20]: wrong section

Comment (JakubFrejlach) [reply docs/CHANGELOG.md:20]: I don't think it really matters here as border between fixed and changed is very thin here. The fix of the validation was more about completely changing it to something else.

Comment (jobselko) [reply docs/CHANGELOG.md:20]: Sorry for not being clear enough - you are updating `4.1.1.`

Comment (JakubFrejlach) [reply docs/CHANGELOG.md:20]: Aha! Thanks :smile:

**Pull request merged**